test/mpi/main.cc

```diff
@@ -574,15 +574,17 @@ void ProcessArgs(int argc, char** argv)
 
 int main(int argc, char *argv[])
 {
-    int failed                                                          = 0;
-    int total_done_skipped_failed[ucc_ilog2(UCC_COLL_TYPE_LAST) + 1][4] = {0};
+    int failed = 0;
+    int total_done_skipped_failed[ucc_ilog2(UCC_COLL_TYPE_LAST) + 1][4];
```

Comment:
I guess unnecessary change for the bug, but does remove a compiler warning.

Reply:
you are right, it just fixes gcc 4.8.5 warning when build ucc_test_mpi